config/sync/views.view.media.yml

```diff
@@ -29,8 +29,12 @@ display:
         options:
           perm: 'access media overview'
       cache:
-        type: tag
-        options: {  }
+        type: time
```

Comment:
Set to time based because if you reference a media item from another place, it doesn't automatically pop the cache item for the view row in the result. It's fairly complex to add these dependencies so a minute's query and render cache TTL is used here to provide a reasonable compromise for now.